proposals/0280-Adding-new-parameter-of-requiresAudioSupport-and-BluetoothDeviceAddress.md

```diff
@@ -31,20 +31,25 @@ If the SDL app or HU cannot use this function, it will perform the same operatio
 1. The developer will set their `requiresAudioSupport` to true.
 2. The library will send a `StartService` for the RPC service with a new param in the payload, `requiresAudioSupport`.
 3. Core will receive the `StartService` for the RPC service: 
-    i. Core will check it's current audio connects (BT A2DP), if it has a supported audio connect it will continue in flow.
-    ii. If there is no Audio support, core will check if it supports the auto connect BT feature. If it does it will continue in flow.
-    iii. If there is no supported audio methods and Core does not support the auto connect BT feature, it will send a `StartServiceNAK` with the reason `No audio support available`.
+
+    1. Core will check its current audio connects (BT A2DP), if it has a supported audio connect it will continue in flow.
+    2. If there is no audio support, Core will check if it supports the auto connect BT feature. If it does, it will continue in flow.
+    3. If there is no supported audio methods and Core does not support the auto connect BT feature, it will send a `StartServiceNAK` with the reason `No audio support available`.
+
 4. If Core has continued, it will send a `StartServiceACK` with a new param `autoBTCapability` set to true.
 5. The app receives the response to its `StartService` for the RPC service:
-    1.If the response was a `StartServiceNAK` the app will shutdown.
-    2. If the response was a `StartServiceACK`, `requiresAudioSupport` was set to true, but the protocol version of the ACK is less than the major version of this feature, the app will shutdown.
-    3. If the response was a `StartServiceACK`, `requiresAudioSupport` was set to true, the protocol version of the ACK is equal or greater than the major version of this feature, and the `autoBTCapability` flag is set to false, the app will check if audio support is available using the `MediaStreamingStatus` class. If it is available it will continue, if not it will shutdown.
+
+    1. If the response was a `StartServiceNAK` the app will shut down.
+    2. If the response was a `StartServiceACK`, `requiresAudioSupport` was set to true, but the protocol version of the ACK is less than the major version of this feature, the app will shut down.
+    3. If the response was a `StartServiceACK`, `requiresAudioSupport` was set to true, the protocol version of the ACK is equal or greater than the major version of this feature, and the `autoBTCapability` flag is set to false, the app will check if audio support is available using the `MediaStreamingStatus` class. If it is available it will continue, if not it will shut down.
```

Comment:
```suggestion
    3. If the response was a `StartServiceACK`, `requiresAudioSupport` was set to true, the protocol version of the ACK is equal to or greater than the major version of this feature, and the `autoBTCapability` flag is set to false, the app will check if audio support is available using the `MediaStreamingStatus` class. If it is available it will continue, if not it will shut down.
```